Tools/Schema.NET.Tool/GeneratorModels/GeneratorSchemaClass.cs

```diff
@@ -50,7 +50,7 @@ public IEnumerable<GeneratorSchemaProperty> DeclaredProperties
             var declaredProperties = this.Properties.Where(classProp => !ancestorProps.Contains(classProp.Name))
                 .OrderBy(x => x.Order);
 
-            return declaredProperties;
+            return declaredProperties.ToList();
```

Comment:
Rather than `ToList`, let's use `ToArray`. It implements `IReadOnlyCollection` and avoids the (admittedly small) allocation overhead of `List`.

Reply:
Done